src/compiler/Parser.js

```diff
@@ -265,6 +265,14 @@ class Parser {
                     }
                 }
 
+                if (attrDef.rawValue) {
+                    let match = /^component\.(?:getE|e)lId\((?:"|')(.*)(?:"|')\)$/.exec(attrDef.rawValue);
```

Comment:
This only supports string literals at the moment.  Also, should walk the AST rather than using a regex here?

Reply:
Yeah I mentioned the string literal bit in my comment above. Additionally, I do think that it's important to walk the AST here too. Technically someone could assign a variable:

```marko
$ var myId = component.getElId('name')

<label for=myId>Name</label>
<input key="name"/>
```